include/cron.php

```diff
@@ -106,8 +107,8 @@ function cron_run(&$argv, &$argc){
 	// Check every conversation
 	ostatus::check_conversations(false);
 
-	// Set the gcontact-id in the item table if missing
-	item_set_gcontact();
+	// Do post update functions
```

Comment:
Mhh difficult. I thought you could say more. I don't know exactly what happens. I interpreted it as something to have some post processing for posts which doesn't have the correct format in the db (im weitesten Sinne) - like the former function item_set_gcontact